common/sgx/endorsements.c

```diff
@@ -382,18 +424,42 @@ oe_result_t oe_get_sgx_endorsements(
 
     // Get quote verification collateral
     OE_CHECK_MSG(
-        oe_get_sgx_quote_verification_collateral_from_certs(
-            &leaf_cert, &quote_verification_collateral),
+        oe_get_sgx_quote_verification_collateral_with_params_from_certs(
+            &leaf_cert, &quote_verification_collateral_with_params),
         "Failed to get certificate quote verification collateral information. "
         "%s",
         oe_result_str(result));
 
+    // Baseline parameters are not used in creating sgx endorsement, in order to
+    // keep oe_create_sgx_endorsements API unchanged, convert
```

Comment:
There's no reason to keep oe_create_sgx_endorsements unchanged since it's a static function in this file, you can just change it as needed, so this phrase does not make sense.